data/campaigns/World_Conquest/scenarios/WC_II_scenario.cfg

```diff
@@ -22,22 +22,89 @@ _ "World Conquest 3p" #enddef
+                experience_penalty={XP}
+            [/value]
+        [/set_variables]
+    [/event]
+#enddef
+
 #define WC2_CAMPAIGN_NEW PLAYERS
-    ## Everything that usually goes into [scenario] like [event], [load_ressource], map_data
-    ## etc is generated by lua. This is also why there is only one [multiplayer] tag: the next
-    ## scenario is again "WC_II_{PLAYERS}p" and the lua code detects that is was called a second
-    ## time and generated the second scenario. (same for third, fourth and fifth). All the lua
-    ## code inside ./lua/map does map/scenario generation.
-    [multiplayer]
+    [campaign]
         id = "WC_II_{PLAYERS}p"
+        define = CAMPAIGN_WC_{PLAYERS}P
         name= {WC_II_CAMPAIGN_NAME_{PLAYERS}P}
         description = {WC_II_CAMPAIGN_DESC_{PLAYERS}P}
-        allow_new_game=yes
-        require_scenario=no
+        first_scenario = "WC_II_{PLAYERS}p"
+        min_players={PLAYERS}
+        max_players={PLAYERS}
+        type = mp
+        abbrev = _ "WC"
+        require_campaign=no
+        {WC2_CAMPAIGN_DIFFICULTY VERY_EASY {WC2_HUMAN_DIFFICULTY human-peasants/peasant purple} {STR_PEASANT} {STR_BEGINNER} 6 2 2 10 yes 0}
+        {WC2_CAMPAIGN_DIFFICULTY EASY {WC2_HUMAN_DIFFICULTY human-loyalists/sergeant black} {STR_SERGEANT} {STR_EASY} 7 3 2 7 yes 5}
+        {WC2_CAMPAIGN_DIFFICULTY NORMAL {WC2_HUMAN_DIFFICULTY human-loyalists/lieutenant brown} {STR_LIEUTENANT} {STR_MEDIUM} 8 4 2 5 yes 10} {DEFAULT_DIFFICULTY}
+        {WC2_CAMPAIGN_DIFFICULTY HARD {WC2_HUMAN_DIFFICULTY human-loyalists/general orange} {STR_GENERAL} {STR_HARD} 8 5 2 2 no 13}
+        # Challenging is a string that exists in mainline but only in campaigns so it probably counts as adding a string?
+        #textdomain wesnoth-httt
+        {WC2_CAMPAIGN_DIFFICULTY VERY_HARD {WC2_HUMAN_DIFFICULTY human-loyalists/marshal white} {STR_GRAND_MARSHAL} _"Challenging" 9 6 2 1 no 17}
+        # Expert is a string that was supposed to be in wesnoth-wc but accidentally got added to wesnoth-units instead
+        #textdomain wesnoth-units
+        {WC2_CAMPAIGN_DIFFICULTY NIGHTMARE {WC2_NIGHTMARE_DIFFICULTY} {STR_NIGHTMARE} _"Expert" 9 7 1 0 no 20}
+    [/campaign]
+#enddef
+
+#define WC2_SCENARIO_NEW TAG ID NAME PLAYERS
+    ## Everything that usually goes into [scenario] like [event], [load_ressource], map_data
+    ## etc is generated by lua. This is also why there is only one [scenario] tag: the next
+    ## scenario is again "WC_II" and the lua code detects that is was called a second
+    ## time and generates the second scenario. (Same for third, fourth and fifth). All the lua
+    ## code inside ./lua/map does map/scenario generation.
+    [{TAG}]
+        id = {ID}
+        name={NAME}
+        allow_new_game=no
         scenario_generation=lua
+        experience_modifier=100
+        victory_when_enemies_defeated=yes
+        carryover_percentage=0
+        carryover_report=no
+        carryover_add=no
+        force_lock_settings=yes
```

Comment:
I moved these from Lua to WML as part of trying to figure out why the hidden sides show… so maybe it's better not to move them.